.gitignore

```diff
@@ -11,3 +11,7 @@ vendor/.git
 node_modules
 vendor
 diff.txt
+.phpcs-cache
+.phpunit.result.cache
+tests/.phpunit.results.cache
```

Comment:
This should not be necessary, see:

https://github.com/joindin/joindin-api/blob/f7dede99cbe0c325565d64626999c4797ef4d68f/phpunit.xml.dist#L11:

Reply:
Ah, I think I still had an unstaged cache file from months back before that was implemented :)

Will drop the commit.